dimod/generators/gates.py

```diff
@@ -13,10 +13,12 @@
 #    limitations under the License.
 
 from collections import defaultdict
-
+from typing import Optional
+import itertools as it
 from dimod.binary.binary_quadratic_model import BinaryQuadraticModel
 from dimod.typing import Variable
 from dimod.vartypes import Vartype
+from dimod import quicksum
```

Comment:
Better to use the full path. Avoids some circular dependency issues as well as making it easier to find for other developers later if needed.

Reply:
Done